foundationdb/src/bin/bindingtester.rs

```diff
@@ -603,7 +603,7 @@ impl StackMachine {
                             key.to_vec().encode(&mut out).expect("failed to encode");
                             value.to_vec().encode(&mut out).expect("failed to encode");
                         }
-                        Ok(out)
```

Comment:
This is unfortunate, not sure if there is a way to get rid of this required type ascription...